### DISCUSSION TITLE
Little(lot) of help please

### DISCUSSION THREAD

**Discussion by Bobabadabouskie** [chuot/rdio-scanner#178] [Q&A] [answered]
I am using sdrTrunk to get audio recordings that are mp3. I run rdio-scanner and tell rdio where the recordings are stored on the config page. Auto populate is turned on everywhere I can find it in the options. The rdio scanner page displays nothing on it's screen. The buttons seem to work but thats it. In the console all I see is this pretty much. No other errors or info: 

```
Rdio Scanner v6.2.5
----------------------------------
2022/05/12 20:47:22 server started
2022/05/12 20:47:22 base folder is /home/dingo/rtl-sdr/rdio
2022/05/12 20:47:22 dirwatches.start: too many open files
2022/05/12 20:47:22 main interface at http://monte:3000
2022/05/12 20:47:22 admin interface at http://monte:3000/admin
2022/05/12 20:47:24 listeners count is 2
2022/05/12 20:47:33 listeners count is 2
2022/05/12 20:48:43 dirwatches.start: too many open files
2022/05/12 20:48:43 configuration changed
2022/05/12 20:51:52 listeners count is 1
```

i will get more of the too many open files error(?) if I let it run longer.

I assume most my problems are due to improper config but I have absolutely no idea how to configure everything, especially the "Systems" tab and all that it includes. 

Usually I can find an example or something to work off of but not with this. Sorry, I have no idea what to do.

Is there a guide or is someone willing to hold my hand(just a little) and walk me through getting started?

I really appreciate any help or assistance. Right now I'm just streaming everything to my phone using icecast and vlc.

**Comment by chuot** [comment, marked as answer]
You are trying to monitor (dirwatch) a folder that most probably has a lot of files in it.

I would recommend to start monitoring an empty folder with a dirwatch that is set to delete the file after it has been ingested. Then start SDR Trunk and you should see the files being ingested (and deleted) on after the other as soon as they're created by SDR Trunk.

The too many open files is also possible if you do have a bunch of app running on the same PC as your Rdio Scanner instance. You may refer to [this page](https://askubuntu.com/questions/1182021/too-many-open-files) to learn how to set a higher value (max files).

**Reply by Bobabadabouskie** [reply]
I emptied the recording folder, set sdrtrunk to record only one talkgroup, delete after ingestion, and rdio still says too many open files.

Looking quickly over the link you posted my ulimit is "unlimited". I don't have time at the moment to explore further but I will and report back.

In the amin interface I see that "no system is defined" even though auto-populate is ticked

Thanks for the help

**Reply by Bobabadabouskie** [reply]
I ended up editing /etc/security/limits.conf and added
```
* soft nofile 65536
* hard nofile 65536
```
to the very bottom and rebooted. This solved the "too many open files" error. Changing things using ulimit didn't work.

Thanks for the tip, i had no idea that was a more general linux error.

It is working now, thank you.
